src/main/rx/crsf.c

```diff
@@ -134,8 +134,7 @@ typedef struct crsfPayloadRcChannelsPacked_s crsfPayloadRcChannelsPacked_t;
 * Uplink is the connection from the ground to the UAV and downlink the opposite direction.
 */
 
-typedef struct crsfPayloadLinkstatistics_s {
-
+typedef struct crsfPayloadLinkstatistics_s{
```

Comment:
If I don't say this, someone else will:
`typedef struct crsfPayloadLinkStatistics_s {`